lib/vue/src/components/input/CInputCheckbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div
+      class="d-flex"
+      style="gap: 1.5rem;"
+    >
+      <h6
+        class="mb-0 text-primary"
+        :class="{ 'text-muted': value }"
+      >
+        No
+      </h6>
+      <b-form-checkbox
+        v-bind="$attrs"
+        v-model="value"
+        switch
+        @input="handleInput"
+      />
+      <h6
+        class="mb-0 strong text-primary"
+        :class="{ 'text-muted': !value }"
+      >
+        Yes
+      </h6>
+    </div>
+
+</template>
+
+<script>
+export default {
+  props: ['value'],
+
+  methods: {
+    handleInput() {
+      this.$emit('input', this.value)
```

Comment:
formatting, for example spaces and so on between () and function title

Also since the event gets the param that has been updated you should use that instead of this.value